README.md

```diff
@@ -51,6 +62,149 @@ server.applyMiddleware({ app })
 
 ```
 
+### Server plugin
+
+Implementation based on server plugin. Implementation entry point is function `validateQuery(schema, query, variables, operationName)`.
```

Comment:
Is there a link to documentation to help provide more context/information on `validateQuery`?

Reply:
Nope, I think it is self-explanatory :-D

Reply:
@velias I'm not sure what this refers to, I'm presuming some low level JavaScript GraphQL API? Would be good to provide more context for those who are more familiar with higher level abstractions

Reply:
Ah, ok. It is my function in `index.js` which is then used to create plugins (Apollo and Envelop). So maybe some better wording should be there, with info that this method can be used to build other plugins?

Reply:
I improved wording to be much clearer (I hope)